### DISCUSSION TITLE
MCD15A2H Assimilation on LIS

### DISCUSSION THREAD

**Discussion by Alirezam1993** [NASA-LIS/LISF#1282] [General]
I am trying to run LAI assimilation using the MODIS MCD15A2H 8 day LAI product on the UMD Zaratan cluster. OL works fine, but here is the place where DA crashes saying that  " ESMF_StateGet failed in generateObservations Stopping." Any recommendation on where to modify the LIS configuration file or the source code?

The log and the config file is attached below. I can see that LIS can reads the LAI retrievals and the climatology files.

[lis.config.txt](https://github.com/NASA-LIS/LISF/files/10771957/lis.config.txt)
[configure.lis.txt](https://github.com/NASA-LIS/LISF/files/10771958/configure.lis.txt)
[lislog.0000.txt](https://github.com/NASA-LIS/LISF/files/10771959/lislog.0000.txt)

**Comment by dmocko** [comment]
HI @Alirezam1993 

Can you please also attach any job standard output/error files?  It might list the routine where the job stopped.

Also, can you please attach your attrib files, as listed here?:

```
Forcing perturbation algorithm:           "none" #"GMAO scheme" 
Forcing perturbation frequency:           "1hr"
Forcing attributes file:                 /home/alirezam/scratch/inputs/forcing_attribs.txt      
Forcing perturbation attributes file:    /home/alirezam/scratch/inputs/forcing_pert_attribs.txt 

State perturbation algorithm:             "none"   #"GMAO scheme"  
State perturbation frequency:             "6hr"
State attributes file:                    /home/alirezam/scratch/inputs/noahmp_lai_attribs.txt
State perturbation attributes file:       /home/alirezam/scratch/inputs/noahmp_lai_pertattribs.txt    

Observation perturbation algorithm:       "none"   #"GMAO scheme"  
Observation perturbation frequency:       "1da"
## edit make/configure.ldt due to error in arch/Config.pl
Observation attributes file:              /home/alirezam/scratch/inputs/PM_LAI_attribs.txt        #./PM_SNWD_attribs.txt          #./PM_DTB1036V_attribs.txt
Observation perturbation attributes file: /home/alirezam/scratch/inputs/PM_LAI_pertattribs.txt    #./PM_DTB_pertattribs.txt    #./PM_SNWD_pertattribs.txt      #./PM_DTB1036V_pertattribs.txt 
```

Finally, can you turn on the `perturbation algorithm` of forcing/state/obs by changing to `"GMAO scheme"`?
